### PR TITLE
Add contributing guidelines and code of conduct

### PR DESCRIPTION
**Pull request by tsalo**
Code of conduct is adapted from [Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html). Contributing guidelines are adapted from [STEMMRoleModels](https://github.com/KirstieJane/STEMMRoleModels).

### CODE REVIEW

**Comment by tsalo** [comment]
@tyarkoni Is there anything else we need to make it easy for any potential first time contributors? Currently, this PR will add a code of conduct and contributing guidelines, which include style preferences (PEP8 + numpydoc, unless you'd prefer something else) for the project. Would a pull request or issues template be useful?

Also, I put your email address in the code of conduct as the point of contact, but I can change it to someone else if you want (e.g., me, @angielaird, or anyone else who's willing and able to fulfill that responsibility).

File: README.md [hunk 1]
```diff
@@ -51,3 +56,8 @@ Once inside the container, you can install NiMARE:
 ```
 python /home/neuro/code/NiMARE/setup.py develop
 ```
+
+## Contributing
+
+Please see our [contributing guidelines](https://github.com/neurostuff/NiMARE/blob/master/CONTRIBUTING.md) for more information on contributing
```
Comment (emdupre) [inline README.md]: Could you also link the Code of Conduct here ?

Comment (tsalo) [reply README.md]: Absolutely. Thanks for taking a look and good catch!

**Comment by tsalo** [comment]
This PR only has minor changes and is ready to merge, I think. @tyarkoni do you mind if I merge it?

**Comment by tyarkoni** [comment]
Fire at will!

**Comment by tsalo** [comment]
Thanks! Hopefully including a code of conduct and contributing guidelines will make it easier for third parties to get involved with NiMARE.

**Pull request merged**